### ISSUE TITLE
Add custom model download path in Settings Tab

### ISSUE THREAD

**Issue by Sniper199999** [Uminosachi/sd-webui-inpaint-anything#82]
A custom path for model download would be great. I use both A1111 and SD.NEXT, and the extension has to download the model file each times for both projects. Having a custom path will solve this issue.

**Comment by Uminosachi** [comment]
Have you considered resolving this issue by creating a symbolic link?

For Windows, you can use:
```bash
mklink <link> <target>
```

Ensure you run the command prompt as an administrator for `mklink` to work.

**Comment by Sniper199999** [comment]
yes, I know this approach, but a more robust method will be to provide a custom path.
